.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+
+[table/table_test.go, *.golden]
```

Comment:
Ideally, we should refactor all table tests to use `.golden` files. Then, we won't need a special rule for `table/table_test.go` anymore.